content/index.ja.md

```diff
@@ -32,187 +33,182 @@ footer: >
 %}
 
 本動画は下記でご覧いただけます：
-* **言語** — [字幕の使用説明書](#translations)は下記の通りです
-* **[画像の説明文](#transcript)**は下記の通りです
-* [YouTube](https://www.youtube.com/watch?v=20SHvU2PKsM)
-* W3C — [MP4、28MB](http://media.w3.org/wai/accessibility-intro/intro.mp4), [日本語字幕 (VTT)](http://media.w3.org/wai/accessibility-intro/W3C_INTRO_SFHI.ja.vtt), [他言語翻訳](http://media.w3.org/wai/accessibility-intro/)
+* **さまざまな言語で** — [字幕やトランスクリプトを見る手順](#translations)は下記の通りです
+* **[ビジュアルの説明を含む文字起こし](#transcript)**は下記の通りです
+* [YouTubeで見る](https://www.youtube.com/watch?v=20SHvU2PKsM)
+* W3Cのサーバで見る — [動画（ファイル形式：MP4、ファイルサイズ：28MB）](http://media.w3.org/wai/accessibility-intro/intro.mp4), [日本語字幕ファイル（VTT）](http://media.w3.org/wai/accessibility-intro/W3C_INTRO_SFHI.ja.vtt), [その他の言語ファイルのリスト](http://media.w3.org/wai/accessibility-intro/)
 {% comment %}# For translations, replace:
 "[English captions file (VTT)](http://media.w3.org/wai/accessibility-intro/W3C_INTRO_SFHI.en.vtt)", with translation of:
 "[[your language] subtitle file (VTT)](http://media.w3.org/wai/accessibility-intro/W3C_INTRO_SFHI.[lang code].vtt)".
 {% endcomment %}
 
-## その他 {#moreinfo}
+## もっと詳しく知る {#moreinfo}
 
-その他詳細：
+詳細については以下を参照：
 
 -   [**ウェブアクセシビリティの紹介**](/fundamentals/accessibility-intro/)
--   [**ウェブアクセシビリティとは**：すべての人に対する影響とその価値](/perspective-videos/) ー本動画と情報は特定のアクセシビリティ要件に特化しています
--   [**WAIウェブサイト**](https://www.w3.org/WAI/)ーウェブアクセシビリティの異なる様々な状況を幅広くサポートするために [標準](/standards-guidelines/), [教育](/teach-advocate/), [テスト／評価](/test-evaluate/), [プロジェクト管理と指針](/planning/).
+-   [**ウェブアクセシビリティの観点からの動画**：すべての人に対する影響とその価値](/perspective-videos/) - 特定のアクセシビリティトピックに関するに情報を動画で紹介します
+-   [**WAIのWebサイト**](https://www.w3.org/WAI/) - Webアクセシビリティのさまざまな側面を幅広くサポートする [標準ガイドライン](/standards-guidelines/)、[教育](/teach-advocate/)、[テスト／評価](/test-evaluate/)、[プロジェクト管理と指針](/planning/)が見つかります。
```

Comment:
@hiroya-uga 

> 標準ガイドライン

My automatic translation tools translates it into "standard guidelines", when the original version only says "standards" since the target page also introduces other standards (WAI-ARIA, WebVTT, etc.) that are not guidelines.

> プロジェクト管理と指針

The original version is "project management, and policy". Can you confirm "policy" is properly translated here?

Reply:
@remibetin I think "standards" means standard guidelines.

@momdo Could I ask you about this problem? Should I use only "標準"？

> The original version is "project management, and policy". Can you confirm "policy" is properly translated here?

umm...
I guess the sentence should be translated as "プロジェクト管理とポリシー".
Because "policy" is already Japanese and the pronunciation is almost same.

But, "指針" also means policy enough.
Should I change it?
I want to know how to translated the word in other page.
I will check it out!

Reply:
@hiroya-uga 

> Could I ask you about this problem? Should I use only "標準"？

I don't know why you chose the translation word "標準ガイドライン", but I think the literal word "標準" is enough.

Reply:
> I guess the sentence should be translated as "プロジェクト管理とポリシー".
Because "policy" is already Japanese and the pronunciation is almost same.

For the word "policy", I could suggest the following translation words:

- "政策" as a government level
- "方針" as an organization level, such as a company